ui/src/components/register/RegisterComponents.tsx

```diff
@@ -0,0 +1,75 @@
+"use client";
+
+import { Box, Paper } from "@mui/material";
+import React from "react";
+import Image from "next/image";
+
+export const MainBox = ({ children }: { children: React.ReactNode }) => (
+    <Box
+        sx={{
+            display: "flex",
+            justifyContent: "center",
+            alignItems: "center",
+            height: "100vh",
+            width: "100vw",
+            backgroundColor: "background.default",
+        }}
+    >
+        {children}
+    </Box>
+);
+
+
+export const RegisterContainer = ({ children }: { children: React.ReactNode }) => (
+    <Box
+        sx={{
+            display: "flex",
+            flexDirection: "row-reverse",
+            width: "800px",
```

Comment:
width and height will have to be update to have dynamic width and heigth, to make website responsive to different sizes of the page.